packages/app-desktop/gui/NoteListItem/utils/useRenderedNote.ts

```
@@ -30,6 +30,8 @@ export default (note: NoteEntity, isSelected: boolean, isWatched: boolean, listR
 	if (listRenderer.dependencies) dependencies = dependencies.concat(listRenderer.dependencies);
 	dependencies = unique(dependencies);
 
+	const previousHashRef = React.useRef<string>('');
```

ESLint

[error] 'React' is not defined.

(no-undef)